src/EnergyPlus/FuelCellElectricGenerator.cc

```diff
@@ -2232,20 +2232,20 @@ namespace FuelCellElectricGenerator {
 
                 // now add energy to storage from charging
                 if ((this->ElecStorage.LastTimeStepStateOfCharge +
-                     tmpPcharge * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour * this->ElecStorage.EnergeticEfficCharge) <
+                     tmpPcharge * state.dataHVACGlobal->TimeStepSys * Constant::SecInHour * this->ElecStorage.EnergeticEfficCharge) <
```

Comment:
For the future, there is TimeStepSys * SecInHour in several places.

Reply:
Right, I don't know why this was not caught. I will `grep` for it again (I rely on `grep` heavily, maybe too heavily, it has definite limitations).